src/components/UserSpellbook.js

```diff
@@ -1,32 +1,32 @@
-import React from "react";
-import dataStub from "../assets/stubData"; //Remove when we get bookmarked spells data from state
+import React, { useContext } from "react";
 import { Link } from "react-router-dom";
 import Spell from "./Spell";
 import Bookmark from "./Bookmark"
+import { UserSpellbookContext } from "../contexts/UserSpellbookContext";
 import "../css/UserSpellbook.css";
```

Comment:
I noticed today that it looks like VSCode wraps the path in double quotes when it auto-imports.  Don't care which one we pick, but it would be good to wrapped them all consistently (that's my favorite word).

Reply:
I always use double-quotes, because then if you need to nest quotes it is possible to do that, but it's not like that comes up very often. It's probably mostly a holdover from fiction writing, I just default to double-quotes.